packages/code-studio/src/settings/FormattingSectionContent.tsx

```diff
@@ -617,7 +617,7 @@ export class FormattingSectionContent extends PureComponent<
                 checked={truncateNumbersWithPound ?? null}
                 onChange={this.handleTruncateNumbersWithPoundChange}
               >
-                Truncate numbers with #
+                Show truncated numbers as ###
```

Comment:
A contextual help bubble could be useful here... I think the phrase "Mask truncated numbers with ###" may be better as well, @dsmmcken any opinion?

Reply:
I am fine with this suggestion. Using the word show is more consistent wit the other labels, like "Show T seperator", and I like the multiple hashtags in the label. It's a bit long now is my only complaint.

Reply:
It only applies to numbers though

Reply:
The other option is to say `Show truncated numbers as #`